app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -214,8 +214,7 @@
       android:textAllCaps="true"
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button1"
       android:visibility="gone"
-      app:backgroundTint="@color/contact_doctor_button"
-      app:iconTint="@android:color/white" />
+      app:backgroundTint="@color/contact_doctor_button" />
```

Comment:
We can stop using this background now, instead we can directly use the `green1` color since we won't have any disable button state now.